improver/standardise.py

```diff
@@ -249,16 +260,16 @@ def process(
         cube = self._rm_air_temperature_status_flag(cube)
         cube = self._collapse_scalar_dimensions(cube)
 
-        if new_name:
-            cube.rename(new_name)
-        if new_units:
-            cube.convert_units(new_units)
-        if coords_to_remove:
-            self._remove_scalar_coords(cube, coords_to_remove)
-        if coord_modification:
-            self._modify_scalar_coord_value(cube, coord_modification)
-        if attributes_dict:
-            amend_attributes(cube, attributes_dict)
+        if self._new_name:
+            cube.rename(self._new_name)
+        if self._new_units:
+            cube.convert_units(self._new_units)
+        if self._coords_to_remove:
```

Comment:
Good software design - all data should be hidden within its class so that users don't accidentally (or purposely) change data owned by the class (hidden in the Python sense by convention, not truly 'hidden').

If you are interested, a recommended reading:
"Object-Oriented Design Heuristics" by Arthur J. Riel

Reply:
Just to feed curiosity, if you wanted to expose this data to users, it would be exposed via setter and getter methods.